x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/rules_list.tsx

```diff
@@ -759,6 +767,34 @@ export const RulesList = ({
     }
   };
 
+  const onEnable = useCallback(async () => {
+    setIsEnablingRules(true);
+
+    const { errors, total } = await bulkEnableRules({
+      ...(isAllSelected ? { filter: getFilter() } : {}),
+      ...(isAllSelected ? {} : { ids: selectedIds }),
+      http,
+    });
+
+    setIsEnablingRules(false);
+    showToast({ action: 'ENABLE', errors, total });
+    await refreshRules();
+  }, [http, selectedIds, getFilter, setIsEnablingRules, toasts]);
```

Comment:
I think `toasts` is not being use in the function, so we can probably remove it from the dependency array.

Maybe you meant to have showToast?